css-text-3/Overview.bs

```diff
@@ -1237,11 +1237,11 @@ Text Processing</h3>
       </ol>
     </li>
     <li><p>If 'white-space' is set to ''pre'', ''pre-wrap'', or ''break-spaces'',
-      any sequence of spaces is treated as a sequence of non-breaking spaces.
+      any sequence of [=preserved white space=] is treated as a sequence of non-breaking spaces.
```

Comment:
This is incorrect. We only do this for spaces, not for tabs.

Reply:
well, we don't do that for tabs that have been preserved as tabs, but tabs (and new lines) that have been converted to spaces and didn't collapse away do count. I was trying to capture that this is not just for U+0020 that have started their life as such, but I guess you're right that tabs that have been preserved as tabs are part of `preserved white space`, and that those are not included.

Possibilities:
* just revert to "spaces", counting on the fact that people will understand that this includes the things that weren't spaces initially but have been converted to spaces at some earlier step. (I'm a little worried about that, because while correct, that stretches the definition of "space" a little bit beyond U+0020, which might make people mistakenly think it includes even more things, such as other space separators).
* rephrase this to make the above point explicit (how?)
* change the definition of `preserved white space` to exclude tabs that have been preserved as tabs, and new lines that have been preserved as new lines, and replace `preserved white space` with "preserved white space, preserved tabs, and preserved newlines" where appropriate.

Reply:
At this point any white space that was previously something else and is converted to a space *is* a space. What it was is not relevant.